### PR TITLE
fix zendesk-ticket-details command

### PR DESCRIPTION
**Pull request by adi88d**
## Status
Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/15788

## Description
When running !zendesk-ticket-details there are attachments listed, but they do not make it into the context data

### CODE REVIEW

File: Releases/LatestRelease/integration-Zendesk.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+- fix zendesk-ticket-details command - attachments dont shows up in the context
+- add test playbook
```
Comment (yaakovi) [inline Releases/LatestRelease/integration-Zendesk.md]: no need to add that to release notes

File: Integrations/integration-Zendesk.yml [hunk 2]
```diff
@@ -901,3 +902,5 @@ script:
     description: Get Zendesk Help Center article
   isfetch: true
 fromversion: 3.5.0
+tests:
+  - No test - tested manually
```
Comment (yaakovi) [inline Integrations/integration-Zendesk.yml]: specify the name of the test playbook

File: Releases/LatestRelease/integration-Zendesk.md [hunk 3]
```diff
@@ -0,0 +1,2 @@
+- fix zendesk-ticket-details command - attachments dont shows up in the context
```
Comment (yaakovi) [inline Releases/LatestRelease/integration-Zendesk.md]: @kirbles19 
please review this release notes

**Comment by kirbles19** [comment]
Done reviewing the .yml.

**Comment by yaakovi** [comment]
@kirbles19 
you removed every empty line in the code...
please avoid doing that next time.

**Pull request merged**